src/NServiceBus.NHibernate/SharedConfig.cs

```diff
@@ -59,14 +59,10 @@ public static PersistenceExtentions<NHibernatePersistence> RegisterManagedSessio
         /// <param name="persistenceConfiguration"></param>
         /// <param name="callback"></param>
         /// <returns></returns>
+        [ObsoleteEx(RemoveInVersion = "8", TreatAsErrorFromVersion = "7", Message = "This is no longer supported. You can still provide your NHibernate Configuration object.")]
```

Comment:
Not clear to me *why* UseCustomSessionCreationMethod isn't supported anymore, or what I should do in case I was counting on this for some reason.

Can you link me to upgrade guide doco / PR that explains this? Even then we should provide a little more detail in the obsolete message.